command: /usr/bin/python /home/arigo/pypysrc/pypy/tool/alarm.py 900 /home/arigo/pypysrc/pypy/interpreter/py.py --oldstyle /home/arigo/pypysrc/lib-python-2.3.4/test/test_ioctl.py
run by: arigo@tismerysoft.de
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Thu Apr 28 00:31:23 2005
pypy-revision: 11518
============================================================
faking <type 'unicode'>
faking <type 'module'>
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
Traceback (application-level):
  File "/home/arigo/pypysrc/lib-python-2.3.4/test/test_ioctl.py", line 2 in ?
from test_support import TestSkipped, run_unittest
  File "test_support.py", line 4 in ?
    raise ImportError, 'test_support must be imported from the test package'
ImportError: test_support must be imported from the test package
========================== closed ==========================
execution time: 25.3406322002 seconds
exit status: 1
